src/app/shared/models/constants/translation-utils.js

```diff
@@ -0,0 +1,48 @@
+const optionsTemplate = (text) => ({
```

Comment:
Remove this if it is not needed

Reply:
@gpalsingh - @mikevic  instructed to add this script.